### ISSUE TITLE
cURL import doesn't parse data binary text correctly

### ISSUE THREAD

**Issue by rmtsrc** [a85/POSTMan-Chrome-Extension#8505]
**Describe the bug**
Importing a cURL command does not decode the text within the `--data-binary` argument correctly when it contains encoded characters.

**To Reproduce**
1. In Chrome DevTools, select the Network panel and right click copy a GraphQL request that uses a required variable and select copy as cURL
2. In Postman click Import > Raw Text and paste the cURL command and Import
3. Click Send
4. The imported cURL request is not the same and the GraphQL service returns an error because the POST body text was not imported correctly from the cURL command

**Expected behavior**
Import cURL command should handle/import `--data-binary` text correctly.

**App information (please complete the following information):**
 - App Type Native App
 - Postman Version 7.24.0
 - OS: macOS Catalina 10.15.4

**Additional context**
Example cURL from Chrome:

```bash
curl 'http://localhost:4000/graphql' \
  -H 'Connection: keep-alive' \
  -H 'Pragma: no-cache' \
  -H 'Cache-Control: no-cache' \
  -H 'accept: */*' \
  -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36' \
  -H 'content-type: application/json' \
  -H 'Origin: https://localhost' \
  -H 'Sec-Fetch-Site: cross-site' \
  -H 'Sec-Fetch-Mode: cors' \
  -H 'Sec-Fetch-Dest: empty' \
  --data-binary $'[{"operationName":"someMutation","variables":{"aRequiredVar":"foo"},"query":"mutation someMutation($aRequiredVar: String\u0021) {\\n  mutateSomething(aRequiredVar: $aRequiredVar) {\\n    message\\n    __typename\\n  }\\n}\\n"}]' \
  --compressed
```

Note that the `!` in `$aRequiredVar: String\u0021` is encoded as `\u0021` and does not get decoded back to `!` when imported.

***Expected Raw Body:***
```json
[
  {
    "operationName": "someMutation",
    "variables": {
      "aRequiredVar": "foo"
    },
    "query": "mutation someMutation($aRequiredVar: String!) {\n  mutateSomething(aRequiredVar: $aRequiredVar) {\n    message\n    __typename\n  }\n}\n"
  }
]
```

***Actual Raw Body:***
```json
[
  {
    "operationName": "someMutation",
    "variables": {
      "aRequiredVar": "foo"
    },
    "query": "mutation someMutation($aRequiredVar: Stringu0021) {\n  mutateSomething(aRequiredVar: $aRequiredVar) {\n    message\n    __typename\n  }\n}\n"
  }
]
```

**Comment by gohumble** [comment]
Same issue here. Any suggestions ?

**Comment by ym500** [comment]
Same Error the body is missing with --data-binary flag

**Comment by VShingala** [comment]
@SebFlippence @gohumble @yosefmahmoud This issue is now fixed with the latest postman app v9.1.2.
